api/src/indigo_savers.cpp

```diff
@@ -545,21 +546,43 @@ CEXPORT int indigoSaveMolfile(int molecule, int output)
     {
         IndigoObject& obj = self.getObject(molecule);
         Output& out = IndigoOutput::get(self.getObject(output));
-
         IndigoSdfSaver::appendMolfile(out, obj);
         out.flush();
         return 1;
     }
     INDIGO_END(-1);
 }
 
-CEXPORT int indigoSaveCml(int item, int output)
+CEXPORT int indigoSaveJson( int item, int output )
 {
     INDIGO_BEGIN
     {
         IndigoObject& obj = self.getObject(item);
         Output& out = IndigoOutput::get(self.getObject(output));
+        if (IndigoBaseMolecule::is(obj))
+        {
+            MoleculeJsonSaver saver(out);
+            BaseMolecule& mol = obj.getBaseMolecule();
+            saver.saveMolecule( mol );
+            out.flush();
+            return 1;
+        }
+        if (IndigoBaseReaction::is(obj))
+        {
+            // reaction not implemented yet
+            return 1;
+        }
+        throw IndigoError("indigoSaveCml(): expected molecule or reaction, got %s", obj.debugInfo());
```

Comment:
`IndigoError("indigoSaveCml()` in function `indigoSaveJson` - copy-paste error.

Reply:
fixed